auth0/src/main/java/com/auth0/android/request/internal/RequestFactory.java

```diff
@@ -137,4 +140,9 @@ AuthenticationRequest createAuthenticationRequest(HttpUrl url, OkHttpClient clie
     Map<String, String> getHeaders() {
         return headers;
     }
+
+    static String getDefaultLocale() {
+        String language = Locale.getDefault().toString();
+        return language != null ? language : "en";
```

Comment:
I think we should check for empty string, according to the docs it says an empty string is returned in case there's no lang or country.